### PR TITLE
MaxBQb.BulkStartStop version 1.0.2

### PR DESCRIPTION
**Pull request by MaxBQb**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103812)

### CODE REVIEW

**Comment by MaxBQb** [comment]
@microsoft-github-policy-service agree

File: manifests/m/MaxBQb/BulkStartStop/1.0.2/MaxBQb.BulkStartStop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MaxBQb.BulkStartStop
+PackageVersion: 1.0.2
+Installers:
+- InstallerUrl: https://github.com/MBQbUtils/BulkStartStop/releases/latest/download/BulkStartStop_setup.exe
```
Comment (Trenly) [inline manifests/m/MaxBQb/BulkStartStop/1.0.2/MaxBQb.BulkStartStop.installer.yaml:7]: Please use the URL to the version and not to Latest, or else SHA mismatches will occur

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/MaxBQb/BulkStartStop/1.0.2/MaxBQb.BulkStartStop.installer.yaml (hunk 1 above)
Comment (MaxBQb) [reply manifests/m/MaxBQb/BulkStartStop/1.0.2/MaxBQb.BulkStartStop.installer.yaml]: fixed

**Comment by MaxBQb** [comment]
Can anyone, please, review my PR?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**